C/Makefile

```diff
@@ -0,0 +1,4 @@
+CFLAGS=$(shell pkg-config --cflags gstreamer-1.0 glib-2.0 gobject-2.0)
```

Comment:
Quick question, Gstreamer has a lot (!) of versions - it's at 1.16 now. It's been a while so I don't remember - are all of these visible to pkg-config while searching using `gstreamer-1.0`?

Reply:
The last zero is only for beauty.
My file gstreamer-1.0.pc contains
Version: 1.14.5

Reply:
Ha! Cool.